qiskit/visualization/pulse_v2/__init__.py

```diff
@@ -11,6 +11,12 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
+
 """
 Pulse visualization module.
 """
+
+from qiskit.visualization.pulse_v2.style.stylesheet import init_style_from_file
+
+
+PULSE_STYLE = init_style_from_file()
```

Comment:
I initially adopted the framework of matplotlib:
https://github.com/matplotlib/matplotlib/blob/57c8baaf85f0cfd44a27ef834dc971128b7f8ee4/lib/matplotlib/__init__.py#L904-L906

but now I lean to hardcord the default settings as python dict rather than json file. I'm just curious about the drawback of coupling to the filesystem.

Reply:
Its harder to test as now we are testing an interaction with the OS. It also means that we can't compile/cache the default configuration (which is not that big a problem). I'm not wholly opposed to this.